features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1145,6 +1145,10 @@
         <!-- Configuration to enable setting local IDP generated AMR values with the values sent by the used idp. -->
         <ReplaceDefaultAMRValuesWithIDPSentValues>{{replace_amr_value_with_idp_sent_values.enable}}</ReplaceDefaultAMRValuesWithIDPSentValues>
 
+        <!-- Configuration to enable role based scope validation. -->
+        <RoleBasedScopeValidation>
```

Comment:
```suggestion
        <RoleBasedScopeIssuer>
```

Reply:
Let's stick into the same name to avoid confusions with default validators

Reply:
Also, this will be better if we add this inside another block
<GlobalScopeValidators>
    <RoleBasedScopeIssuer>
        <Enable></Enable>
    </RoleBasedScopeIssuer>
</GlobalScopeValidators>

Reply:
Resolved with cc32814cf8d167e01b97c64ad0a7632e939904ba